test/integration_serial_test.go

```diff
@@ -112,7 +109,7 @@ func TestTerragruntCorrectlyMirrorsTerraformGCPAuth(t *testing.T) {
 	defaultCreds := os.Getenv("GCLOUD_SERVICE_KEY")
 	defer os.Setenv("GCLOUD_SERVICE_KEY", defaultCreds)
```

Comment:
`os.Setenv` is still used, to avoid this I think can be used t.Setenv(GCLOUD_SERVICE_KEY, "")

Reply:
Fixed. Also rebased on `master`.